site/docs/features/management.md

```diff
@@ -99,7 +99,7 @@ The above snippet assumes a `TABLE_IDENTIFIER` which points to the unreferenced
 spark session and a nessie owned `Catalog`. The `result` object above returns the number of files the action tried to delete and the number that failed.
 
 !!! note
-    You can follow along an interactive demo in a [Jupyter Notebook via Google Colab](https://colab.research.google.com/github/projectnessie/nessie-demos/blob/main/colab/nessie-iceberg-demo-nba.ipynb)
+    You can follow along an interactive demo in a [Jupyter Notebook via Google Colab](https://colab.research.google.com/github/projectnessie/nessie-demos/blob/main/notebooks/nessie-iceberg-demo-nba.ipynb)
```

Comment:
This should point to the Binder notebook, we no longer support Google Colab.

Reply:
yeah, below this line it has binder link as well. I shall remove colab link then.

Reply:
@snazy :  what about links to this (nessie-iceberg-spark-setup.ipynb) notebook ? I saw that recently during clean up this document is deleted. So, I will remove entire text talking about this document ?

Reply:
No, don't delete it. Just replace the link with
```suggestion
    You can follow along an interactive demo in [this Notebook via Binder](https://mybinder.org/v2/gh/projectnessie/nessie-demos/main?filepath=notebooks/nessie-iceberg-demo-nba.ipynb)
```

Reply:
ok. I have changed as per your suggestion